idaes_examples/mod/tut/visualizer_tutorial.py

```diff
@@ -289,7 +290,13 @@ def initialize_model(m: ConcreteModel) -> ConcreteModel:
     seq.set_guesses_for(m.fs.H101.inlet, tear_guesses)
 
     def initialize_unit(unit):
-        unit.initialize(outlvl=idaeslog.DEBUG)
+        try:
+            initializer = unit.default_initializer()
+            initializer.initialize(unit, output_level=idaeslog.INFO)
+        except:
```

Comment:
Can we identify the specific exception you expect here (I suspect it is an `AttributeError`).

It is generally bad practice to use bare `except` statements as that will handle any exception that occurs which can hide unexpected behaviors,